chrysler_pacifica_ehybrid_s_2019/VehicleConfigParams.yaml

```diff
@@ -94,6 +94,9 @@ lidar_gps_drivers:
 camera_drivers:
   - /hardware_interface/camera
 
+# Vehicle Participation Type
```

Comment:
These comments should provide a bit more detail then just a duplication of the variable name
How about
"Vehicle Participation Type which corresponds to the Lanelet2 participant type standards. Used for interpreting traffic rules in the world model"